src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const FORMAT_SOURCE = sourceString.split('\n').join('');
+  const STYLE_ARRAY = FORMAT_SOURCE.split(';');
+
+  const STYLE_OBJECT = {};
+
+  const PAIRS_OF_STYLE = STYLE_ARRAY.map((item) => item.split(':'));
+
+  PAIRS_OF_STYLE.forEach((item) => {
+    if (item.length < 2) {
+      return 0;
+    }
+
+    const KEY = item[0].trim();
+    const VALUE = item[1].split(',').join(`,\n`).trim();
```

Comment:
The split and join operations here are unnecessary since the task does not require handling multiple values separated by commas differently. They can be removed to simplify the code.